build_runner/lib/src/generate/build_impl.dart

```diff
@@ -221,6 +221,7 @@ class _SingleBuild {
       _logger.severe('Failed after ${humanReadable(watch.elapsed)}',
           result.exception, result.stackTrace);
     }
+    print('-----------------------------------------------------------------');
```

Comment:
We'll need to go through the `_logger` rather than `print`. We also only want this for the watch/serve case, not for every build.

Will add a similar line with https://github.com/dart-lang/build/pull/1276